projects/analytics_logging/resources/templates/fastly_user.tpl

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Is there a reason you're not using a [policy doc](terraform.io/docs/providers/aws/d/iam_policy_document.html) for this?